app/dashboard/templates/bounty_details.html

```diff
@@ -88,6 +88,12 @@ <h4 id="title" class="font-title"></h4>
                   <span class="bounty-info-text" id="project_type" title='{% include "shared/project_type_tooltip.html" %}'>></span>
                   <i class="fas fa-info-circle" title='{% include "shared/project_type_tooltip.html" %}'></i>
                 </div>
+
+                <div>
+                  <span class="bounty-info-heading">{% trans "Project Length" %}</span>
```

Comment:
@PixelantDesign Should I change "Project Length" to "Project Timeframe"?

Reply:
I guess this isn't needed as @PixelantDesign has approved the changes! ^_^